arcosh.json

```diff
@@ -1,29 +1,31 @@
 {
     "id": "arcosh",
     "summary": "Inverse hyperbolic cosine",
-    "description": "Computes the inverse hyperbolic cosine of `x`. It is the inverse function of the hyperbolic cosine so that *`arcosh(cosh(x)) = x`*.\n\nWorks on radians only.\nThe no-data value `null` is passed through and therefore gets propagated.",
+    "description": "Computes the inverse hyperbolic cosine of `x`. It is the inverse function of the hyperbolic cosine so that *`arcosh(cosh(x)) = x`*.\n\nWorks on radians only.\nThe no-data value `null` is passed through and therefore gets propagated. `NaN` is returned for values outside of the allowed range.",
```

Comment:
> Works on radians only.

I have never seen the explicit clarification of "radians" in context of hyperbolic trig functions. I don't think there is an alternative definition like you have radians vs degrees with regular trig functions. It looks confusing to me to read that "Works on radians only." and made me double check I wasn't reading about standard cosine function

Reply:
I guess that was copy pasted back in the days. Removed.